tests/conftest.py

```diff
@@ -36,3 +37,15 @@ def clear_cache():
     from conda.core.subdir_data import SubdirData
 
     SubdirData._cache_.clear()
+
+
+@pytest.fixture()
+def support_file_server():
+    """
+    Open a local web server to test remote support files.
+    """
+    base = Path(__file__).parents[0] / "conda_env" / "support"
+    port = "8928"   # randomly chosen
```

Comment:
`randint(8000,9000)`

Reply:
If it becomes a problem we will use an os-chosen port; but `http.server` has a more convenient CLI than Python interface AFAICT. We could tell it to listen on port 0 and try to parse its stdout, but this is always a challenge since you could wait forever on unflushed i/o.